addons/calendar_sms/views/calendar_views.xml

```diff
@@ -8,7 +8,7 @@
         <field name="view_mode">form</field>
         <field name="target">new</field>
         <field name="context">{
-            'default_composition_mode': 'guess',
+            'sms_composition_mode': 'guess',
```

Comment:
Initial comment:
Don't change that, this is a "keyword" used notably in sms composer.